### PR TITLE
s/network-tracer/system-probe

### PR DESCRIPTION
**Pull request by sfluor**
### What does this PR do?

Renames the `network-tracer` into `system-probe`

### Motivation

Better name for the `network-tracer`

### CODE REVIEW

File: Dockerfiles/agent/s6-services/network/finish
Comment (sunhay) [inline Dockerfiles/agent/s6-services/network/finish]: Does it make sense to move these to a different directory? e.g. `Dockerfiles/agent/s6-services/system-probe/*`

File: cmd/system-probe/main.go
Comment (sunhay) [inline cmd/system-probe/main.go]: `SYS-PROBE`?

**Review by sunhay — DISMISSED** [review]
Overall looks good to me - we'll need to change our internal daemon set + chef recipes to handle this as well

**Review by olivielpeau — DISMISSED** [review]
Reviewed the changes on the invoke tasks and the packaging, LGTM if you've confirmed the gitlab pipeline on your branch passes 😄

**Review by sunhay — DISMISSED** [review]

**Review by hkaj — DISMISSED** [review]
LGTM, you may want to update CODEOWNER and the .gitignore as well.

**Review by hkaj — APPROVED** [review]

**Review by sunhay — APPROVED** [review]

**Review by olivielpeau — APPROVED** [review]

**Pull request merged**
